lib/ex_cell/ex_cell.ex

```diff
@@ -1,5 +1,6 @@
 defmodule ExCell do
   @moduledoc false
```

Comment:
Don't we want to include these in the docs? Or laziness? :D

Reply:
Moved functions here. Previously these were documented elsewhere.